build.yaml

```diff
@@ -1584,6 +1584,17 @@ steps:
            --verbose \
            site/www/* \
            /io/www
+
+     # Grab the latest release of the docs
+     LATEST_02_SHA=$(gsutil cat gs://hail-common/builds/0.2/latest-hash/cloudtools-5-spark-2.4.0.txt 2>/dev/null)
```

Comment:
Should we ensure it's not the empty string?

Reply:
Sure. What would be the reaction to it being empty? Fail in this script?

Reply:
I think so.